packages/core/src/transport-http.ts

```diff
@@ -147,9 +148,9 @@ export class StreamableHttpTransport {
           return;
         }
         const allowedCrossSessionNotifications = [
-          NOTIFICATIONS.TOOLS_LIST_CHANGED,
-          NOTIFICATIONS.PROMPTS_LIST_CHANGED,
-          NOTIFICATIONS.RESOURCES_LIST_CHANGED,
+          METHODS.NOTIFICATIONS.TOOLS.LIST_CHANGED,
+          METHODS.NOTIFICATIONS.PROMPTS.LIST_CHANGED,
+          METHODS.NOTIFICATIONS.RESOURCES.LIST_CHANGED,
```

Comment:
this seems like its own constant - which would also help clean up the `method as (typeof allowedCrossSessionNotifications)[number]` type cast later in the file

Reply:
created new comment for updated code, closing this one